apps/user-profile/src/components/modules/Onboarding/UserLandingPage/UserLandingPage.js

```diff
@@ -133,16 +138,20 @@ export default function UserLandingPage() {
     setPerson({ ...person, lastName: e.target.value, lastNameTouched: true });
   }
 
-  function onLocationChange(e) {
-    setPerson({ ...person, location: e.target.value });
+  function onCountryChange(e) {
+    setPerson({ ...person, country: e });
   }
 
   function onRoleChange(e) {
     setPerson({ ...person, role: e.target.value });
   }
 
-  function onHeadLineChange(e) {
-    setPerson({ ...person, headline: e.target.value });
+  function onLinkedInPortfolioChange(e) {
+    setPerson({ ...person, linkedInPortfolio: e.target.value });
+  }
+
+  function onWebsitePortfolioChange(e) {
+    setPerson({ ...person, websitePortfolio: e.target.value });
   }
```

Comment:
Can you comment these out, don't delete it, just comment it out. This won't be going into the November release until the backend has been sorted out.